scripts/e2e/deploy-merlin.sh

```diff
@@ -23,7 +23,6 @@ install_mlp() {
     --set mlp.ingress.enabled=true \
     --set mlp.ingress.class=istio \
     --set mlp.ingress.host=mlp.mlp.${INGRESS_HOST} \
-    --set mlp.ingress.path="/*" \
```

Comment:
why do we remove this?

Reply:
Because https://github.com/gojek/mlp/pull/67 the path is `PrefixType` and shouldn't contain wildcard